daisy_workflows/image_build/enterprise_linux/ks_helpers.py

```diff
@@ -276,6 +264,17 @@ def BuildKsConfig(release, google_cloud_repo, byol, sap, sap_hana, sap_apps,
     custom_post = '\n'.join([ol_post, el_post])
     cleanup = FetchConfigPart('el7-cleanup.cfg')
     repo_version = 'el7'
+  elif release == "rhel8":
+    logging.info('Building RHEL 8 image.')
+    ks_packages = FetchConfigPart('el8-packages.cfg')
+    ks_options = FetchConfigPart('el8-options.cfg')
+    if uefi:
+      ks_options = FetchConfigPart('el8-uefi-options.cfg')
+    rhel_post = FetchConfigPart('rhel8-post.cfg')
```

Comment:
is it intentional to have `el8-post` AND `rhel8-post` files? can they be more descriptively named?

Reply:
It is- that's because EL8 will apply to CentOS as well, where as rhel8 only applies to RHEL.